windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
So you can just double click the project file and it opens in Visual Studio 2013 just fine? That's news to me. Did it prompt you that the project could not be loaded? Did it prompt you to convert the project to a compatible VS version? Did the project file change to v120 everywhere after opening it?